code/ImageGalleryPage.php

```diff
@@ -157,6 +157,8 @@ public function getCMSFields() {
 				)
 			);
 		}
+
+		$this->extend('updateCMSFields', $fields);
```

Comment:
Some extensions don't know that they are being called a second time.

The solution is to use `beforeUpdateCMSFields`

See http://doc.silverstripe.org/en/developer_guides/extending/extensions/#object-extension-injection-points